pkg/controller/cluster/agent/shared.go

```diff
@@ -26,18 +26,20 @@ const (
 )
 
 type SharedAgent struct {
-	cluster          *v1alpha1.Cluster
-	serviceIP        string
-	sharedAgentImage string
-	token            string
+	cluster                    *v1alpha1.Cluster
+	serviceIP                  string
+	sharedAgentImage           string
+	sharedAgentImagePullPolicy string
```

Comment:
We can omit the redundant prefix:

```suggestion
	image           string
	imagePullPolicy string
```

Reply:
okidoke